tests/dummy/app/pods/demo/template.hbs

```diff
@@ -32,16 +32,17 @@
     yScale=(d3-linear-scale)
     as |chart|
   }}
-    {{#chart.x-axis label='Line' as |tick|}}
-      {{to-fixed tick 0}}
-    {{/chart.x-axis}}
+    {{#chart.x-axis-svg label='Line' alignment='top' offset=-6 as |tick|}}
```

Comment:
Any clue as to why the `-6` offset is necessary?  Can we raise an issue for this?